k8s/0.26.2/eks/vsecm-distroless-fips.yaml

```diff
@@ -95,6 +97,8 @@ metadata:
     kubernetes.io/enforce-mountable-secrets: "true"
     kubernetes.io/mountable-secrets: vsecm-sentinel-init-secret
 automountServiceAccountToken: false
+secrets:
+  - name: vsecm-sentinel-init-secret
```

Comment:
It feels like repetitive since the above annotation already has the secret's name; but without it I get errors and the statefulset did not initialize — so I'm adding it nonetheless.

Also k8s docs say it's a good practice to be explicit about these kinds of secrets.